### PR TITLE
Implicit nullable types for PHP 8.4 compatibility

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Implicit nullable types for PHP 8.4 compatibility
| Type?             | refactor
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Sponsor company   | 
| How to test?      |

### CODE REVIEW

**Review by jolelievre — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
